ang/civicase/activity/services/activity-feed-measurements.service.js

```diff
@@ -0,0 +1,19 @@
+(function ($, angular) {
+  var module = angular.module('civicase');
+
+  module.service('ActivityFeedMeasurements', function () {
+    this.getTopOffset = getTopOffset;
+
+    /**
+     * Returns the offset of feed body from top
+     *
+     * @return {Number}
+     */
+    function getTopOffset () {
+      var $feedBody = $('.civicase__activity-feed__body');
+      var topOffset = $feedBody.offset().top + 24; // 24px is the padding-top of `panel-body` of the feed
```

Comment:
why don't you use jQuery to find out 24 anyways?

If for some reason this is not possible this is not possible, then move the 24 to a new constant like var: `PANEL_BODY_PADDING_TOP` maybe?

Reply:
Good idea, using jquery to calculate the padding.